src/store/sagas/digitalAssets/index.js

```diff
@@ -2,23 +2,23 @@
 
 import {
   digitalAssetsRootSaga,
-} from './digitalAssets'
+} from './core'
 
 import {
   addAssetRootSaga,
-} from '../routes/AddAsset/sagas/addAsset'
+} from './addAsset'
 
 import {
   editAssetRootSaga,
-} from '../routes/EditAsset/sagas/editAsset'
+} from './editAsset'
 
 import {
   digitalAssetsGridRootSaga,
-} from '../routes/Grid/sagas/digitalAssetsGrid'
+} from './digitalAssetsGrid'
 
 import {
   digitalAssetsSendRootSaga,
-} from '../routes/Send/sagas/digitalAssetsSend'
+} from './digitalAssetsSend'
 
```

Comment:
I suggest such changes:
```
import { digitalAssetsCoreRootSaga } from './core'
import { digitalAssetsAddRootSaga } from './add'
import { digitalAssetsEditRootSaga } from './edit'
import { digitalAssetsGridRootSaga } from './grid'
import { digitalAssetsSendRootSaga } from './send'

export default {
  digitalAssetsCoreRootSaga,
  digitalAssetsAddRootSaga,
  digitalAssetsEditRootSaga,
  digitalAssetsGridRootSaga,
  digitalAssetsSendRootSaga,
}
```

Reply:
This exports already in digitalAssets namespace, I guess the name like `coreRootSaga`, `editRootSaga` is enough